### DISCUSSION TITLE
PAC as dotnet tool

### DISCUSSION THREAD

**Discussion by rajyraman** [microsoft/powerplatform-vscode#478] [Ideas]
I was reading though https://github.com/MicrosoftDocs/power-platform/issues/1835

Since PAC is being distributed through NuGet, it would be good if pac is released as a [dotnet tool](https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools-how-to-create) as well.

**Comment by tehcrashxor** [comment]
Latest release of PAC has also been released as a dotnet tool - [Microsoft.PowerApps.CLI.Tool](https://www.nuget.org/packages/Microsoft.PowerApps.CLI.Tool)

Note that the VS Code extension that this repo covers is not yet updated to replace our packing of linux-x64 and osx-x64 versions, as the change came in too late for the release cutoff, but will _likely_ be in next month's via #529
